docs/09-Configuration reference/02-Custom Resource Definitions.md

```
@@ -2177,6 +2178,95 @@ GatewayHTTPAPI is the Schema for the HTTPAPIs API
+
+
+
+
+
+
+
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `stackSelector` _[LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#labelselector-v1-meta)_ |  |  |  |
+| `traces` _[OtelSignalConfig](#otelsignalconfig)_ |  |  |  |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)